site/e2e/playwright.config.ts

```diff
@@ -55,7 +54,7 @@ export default defineConfig({
       `--access-url=http://localhost:${coderPort}`,
       `--http-address=localhost:${coderPort}`,
       // Adding an enterprise license causes issues with pgcoord when running with `--in-memory`.
-      !enterpriseLicense && "--in-memory",
+      "--in-memory",
```

Comment:
prolly don't need the comment above this

Reply:
Good catch. :+1: